packages/backend-plugin-api/api-report.md

```diff
@@ -24,6 +25,64 @@ import { Readable } from 'stream';
 import { Request as Request_2 } from 'express';
 import { Response as Response_2 } from 'express';
 
+// @public
+export type AuditEvent = [
+  message: string,
+  meta: {
+    actor: AuditEventActorDetails;
+    eventName: string;
+    stage: string;
+    meta?: JsonValue;
+    request?: AuditEventRequest;
+  } & AuditEventStatus,
+];
+
+// @public (undocumented)
+export type AuditEventActorDetails = {
+  actorId?: string;
+  ip?: string;
+  hostname?: string;
+  userAgent?: string;
+};
+
+// @public
+export type AuditEventArgs = {
+  message: string;
+  eventName: string;
+  stage: string;
+  request?: Request_2;
+  actorId?: string;
```

Comment:
Is this one here to be able to override it or something? Or is it to be used when there's no request present? Because it seems that there's a bit of a duality here with the request and the actor. Either way, should it be `actor?: AuditEventActorDetails` instead?

Reply:
This was included to allow an actor, already known in advance, to be reused across multiple logging events, optimizing compute resources. However, it can be removed for the sake of simplicity.